src/Raven.Server/Documents/ETL/EtlTransformer.cs

```diff
@@ -164,6 +168,64 @@ private JsValue LoadCounter(JsValue self, JsValue[] args)
             return loadCounterReference;
         }
 
+        private JsValue LoadTimeSeries(JsValue self, JsValue[] args)
+        {
+            const int paramsCount = Transformation.TimeSeriesTransformation.LoadTimeSeries.ParamsCount;
+            const string signature = Transformation.TimeSeriesTransformation.LoadTimeSeries.Signature;
+            
+            if (args.Length == paramsCount)
+                ThrowInvalidScriptMethodCall($"{signature} must have {paramsCount} arguments");
+                
+            if(args[0].IsString() == false)
+                ThrowInvalidScriptMethodCall($"{signature}. The argument timeSeriesName must be string");
+            var timeSeriesName = args[0].AsString();
+
+            var from = GetDateArg(args[1], signature, "from"); 
+            var to = GetDateArg(args[2], signature, "to"); 
+                
+            JsValue loadCounterReference = (JsValue)Transformation.TimeSeriesTransformation.Marker + timeSeriesName;
+
+            if ((Current.Document.Flags & DocumentFlags.HasTimeSeries) == DocumentFlags.HasTimeSeries)
+            {
+                var reader = Database.DocumentsStorage.TimeSeriesStorage.GetReader(Context, Current.DocumentId, timeSeriesName, from, to);
+                //TODO Maybe to return null if no entries
```

Comment:
You aren't handling tombstones here, are you handling that elsewhere?

Reply:
In progress